src/EntityFramework/Core/Common/EntitySql/SemanticAnalyzer.cs

```diff
@@ -3967,7 +3968,8 @@ private static DbExpressionBinding ProcessGroupByClause(DbExpressionBinding sour
             if (null != queryExpr.HavingClause
                 && queryExpr.HavingClause.HasMethodCall)
             {
-                var converted = ConvertValueExpression(queryExpr.HavingClause.HavingPredicate, sr);
+                // LAJ LAJ - needed for side-effects?
+                ConvertValueExpression(queryExpr.HavingClause.HavingPredicate, sr);
```

Comment:
In other versions of VS this may cause a different CodeAnalysis error where we ignore the result of a method call. If that's the case I'll suppress that message.